envs/nick_gym_adapter.py

```diff
@@ -5,6 +5,10 @@
 
 
 class Nick2048Gym(Nick2048):
+
+    def __init__(self, config=None, random_seed=None):
+        super().__init__(config, random_seed)
```

Comment:
I think it might make sense for the adapter to unpackage the config here and pass the random seed (if it exists), rather than having `Nick2048` know about two ways to set the random seed.

Reply:
Done.